src/user_prompts.rs

```diff
@@ -0,0 +1,35 @@
+//! User prompt related functionality for WebDriver.
+//!
+//! See [18. User Prompts](https://www.w3.org/TR/webdriver1/#user-prompts) of the WebDriver
+//! standard.
+use webdriver::command::WebDriverCommand;
+
+use crate::error;
+use crate::Client;
+
+/// `UserPrompt` enumerates the different ways a `Client` can respond to a user prompt, or alert in
+/// the browser window.
+#[derive(Debug, Clone)]
+pub enum UserPrompt {
```

Comment:
This should be marked `#[non_exhaustive]` so that we can add more variants (like `GetText`) later on without a breaking release.

Reply:
Also, I wonder if this should be named `PromptAction` rather than `UserPrompt`, since it represents an action, not the prompt itself.

Reply:
Done.